galera/src/replicator_smm.cpp

```diff
@@ -3158,9 +3159,18 @@ void galera::ReplicatorSMM::process_prim_conf_change(void* recv_ctx,
             // was not part of IST, don't discard
             cc_buf_discard.keep(conf.seqno);
         }
+
+        /* 5.7 does not generate new seqno for CC, so if we are in the
+           mixed version cluster (5.7 and 8.0 nodes) and nodes are:
+           node_1 (5.7), node_2 (8.0 - this node), node_3 (8.0 - just joining)
+           and there was no any DDL/DML between node_2 and node_3 join,
+           during node_3 join we still see the same conf.seqno as it was
+           recorded during node_2's SST. */
+        if (conf.seqno == sst_seqno_ && mixed_ver_cluster) {
+            submit_ordered_view_info(recv_ctx, view_info.get());
+        }
```

Comment:
Here and in the next chunk. Please use galera style formatting.
```suggestion
        if (conf.seqno == sst_seqno_ && mixed_ver_cluster) 
        {
            submit_ordered_view_info(recv_ctx, view_info.get());
        }
```

Reply:
done